app/policies/feed_policy.rb

```diff
@@ -1,4 +1,8 @@
 class FeedPolicy < ApplicationPolicy
+  def show?
+    update?
```

Comment:
Is this true, though?  What about readonly users?  I think you want to check if the user can `show?` the Podcast.

Reply:
FWIW, I just gave the ID staging user `ryancavistest` a readonly role.  So we can masquerade as that, (and click the "reload" button in the Feeder upper-right dropdown), and test some of this readonly stuff.

(I'm actually seeing some issues throughout the app with readonly ... so maybe we'll end up with a ticket to just go through and fix all the pages).